app/elements/todo-view/todo-view.html

```diff
@@ -214,6 +214,10 @@
         if (drawerPanel.narrow) {
           drawerPanel.closeDrawer();
         }
+      },
+
+      signOut: function() {
+        document.querySelector("todo-data").signOut();
```

Comment:
You wouldn't want todo-view to reach outside of itself and affect another element. It shouldn't know what its data provider is, otherwise they're tightly coupled.

I implemented basic sign out here: https://github.com/PolymerLabs/todo-list/commit/1d2a90f232b0e6f9cea560c0b8a38967bad355f9  When someone clicks the button in todo-view it fires an event which gets handled by the common parent of todo-view and todo-data. That parent acts as a mediator and decides to call a method on todo-data based on that event.